src/assets/SASSAsset.js

```diff
@@ -38,7 +38,9 @@ class SASSAsset extends Asset {
       }
     });
 
-    opts.importer = (url, prev, done) => {
+    opts.importer = opts.importer || [];
+    opts.importer = Array.isArray(opts.importer) ? [].concat(opts.importer) : [opts.importer];
```

Comment:
Why is `[].concat(opts.importer)` necessary, shouldn't it be safe to append to this array?

Reply:
oh my bad! This might be some remaining of my testing code... I'll fix it 👍